<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xyzInchingWidget</class>
 <widget class="QWidget" name="xyzInchingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>473</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>InchingWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="controlBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QPushButton" name="startBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/blue_starticon.png</normaloff>
            <normalon>:/icons/blue_stopicon.png</normalon>:/icons/blue_starticon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="backwardBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/backward.png</normaloff>:/icons/backward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="clockwiseBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/clockwise.png</normaloff>:/icons/clockwise.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="counterClockwiseBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/counterclockwise.png</normaloff>:/icons/counterclockwise.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="rightBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/rightward.png</normaloff>:/icons/rightward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="leftBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/leftward.png</normaloff>:/icons/leftward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="upBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/upward.png</normaloff>:/icons/upward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="forwardBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/forward .png</normaloff>:/icons/forward .png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="downBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>70</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/downward.png</normaloff>:/icons/downward.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>65</width>
            <height>65</height>
           </size>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="rvelSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rvelSpinBox">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>deg/s</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QSlider" name="lvelSlider">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lvelSpinBox">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>mm/s</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusBox">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="axisXLabel">
          <property name="text">
           <string>AxisX:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit">
          <property name="text">
           <string>mm/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="axisYLabel">
          <property name="text">
           <string>AxisY:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="yPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_3">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="yVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_3">
          <property name="text">
           <string>mm/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="axisZLabel">
          <property name="text">
           <string>AxisZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_2">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_2">
          <property name="text">
           <string>mm/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="axisRLabel">
          <property name="text">
           <string>AxisR:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="rPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_4">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="rVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_4">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>rvelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rvelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rvelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rvelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lvelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lvelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
